pysqldb3/geopackage.py

```diff
@@ -171,38 +228,95 @@ def shp_to_gpkg(self, shp_name=None, gpkg_name=None, tbl_name=None, print_cmd=Fa
 
         return
 
-    def gpkg_to_shp(self, export_path = None, gpkg_name=None, print_cmd=False):
+    def gpkg_to_shp(self, print_cmd=False):
```

Comment:
this should take in an output path for the shape, it can default back to the same path as the geopackage, but that should be over writeable